src/Kdyby/Doctrine/Console/CommandHelper.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/**
+ * This file is part of the Kdyby (http://www.kdyby.org)
+ *
+ * Copyright (c) 2008 Filip Procházka (filip@prochazka.su)
+ *
+ * For the full copyright and license information, please view the file license.txt that was distributed with this source code.
+ */
+
+namespace Kdyby\Doctrine\Console;
+
+use Doctrine\DBAL\Tools\Console\Helper\ConnectionHelper;
+use Doctrine\ORM\Tools\Console\Helper\EntityManagerHelper;
+use Kdyby\Console\ContainerHelper;
+use Nette;
+
+
+
+/**
+ * @author Tomáš Jacík <tomas@jacik.cz>
+ */
+abstract class CommandHelper extends Nette\Object
+{
+
+	public static function setApplicationEntityManager(ContainerHelper $containerHelper, $emName)
```

Comment:
Yes, but that is for the purposes of the `Application` class, getting the `Container` from it would add a level of indirection and would violate DI principles.